code/modules/projectiles/boxes_magazines/ammo_stacks/prefab_stacks/premade_pistol_stacks.dm

```diff
@@ -490,7 +492,7 @@
 /obj/item/storage/box/ammo/a44roum_hp
 	name = "box of HP .44 roumain ammo"
 	desc = "A box of .44 roumain rubbershot ammo, designed to disable targets without causing serious damage."
-	icon_state = "a44roum-rubber"
+	icon_state = "a44roum-hp"
```

Comment:
that doesn't seem right. is there no sprite for `-rubber` here?

Reply:
This isn't rubber. This is hollow point.

Reply:
ooprs